fastlane/Fastfile

```diff
@@ -0,0 +1,87 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+fastlane_require 'semantic'
+
+default_platform(:android)
+
+platform :android do
```

Comment:
Not sure what difference it makes, but is `android` the correct platform to use here?

Reply:
Pretty sure we need to nominate one platform, but in this case it doesn't matter as it's just for the tagging